source/includes/_chat_summary.md

```diff
@@ -357,6 +520,7 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 |campaign_start_date|String|Start date of campaign|
 |campaign_end_date|String|End date of campaign|
 |intervals|Object|Data for specified interval. [Intervals Object](#chatintervals)|
+<internal>|web_publisher_campaigns|Array|array of web_publisher_campaign objects [Web Publisher Campaign Object](#webpublishercampaign)|</internal>
```

Comment:
Move the tag inside the table row: `|<internal>web_publisher_campaigns|Array|array of web_publisher_campaign objects [Web Publisher Campaign Object](#webpublishercampaign)</internal>|`

Reply:
Fixed